man/man8/zpool.8

```diff
@@ -2045,6 +2063,17 @@ Because scrubbing and resilvering are \fBI/O\fR-intensive operations, \fBZFS\fR
 .RS 6n
 Stop scrubbing.
 .RE
+.sp
+.ne 2
+.na
+\fB\fB-p\fR\fR
+.ad
+.RS 6n
+Pause scrubbing. Scrub progress is periodically synced to disk so if the system
+is restarted or pool is exported during a paused scrub, the scrub will resume
+from the place where it was last checkpointed to disk. To resume a paused scrub
+issue \fBzpool scrub\fR again.
```

Comment:
I'd be nice to avoid the feature flag and reserve a `dsl_scan_flags_t` bit for this to avoid potential GRUB compatibility issues.